### PR TITLE
Take options (ex: boost, fuzziness) into consideration in NestedQueryBackend

### PR DESCRIPTION
**Pull request by hemache**

### CODE REVIEW

File: src/django_elasticsearch_dsl_drf/filter_backends/search/query_backends/nested.py [hunk 1]
```diff
@@ -75,15 +75,21 @@ def construct_search(cls, request, view, search_backend):
                     for _field in options.get('fields', []):
                         # In case if we deal with structure 2
                         if isinstance(_field, dict):
-                            # TODO: take options (ex: boost) into consideration
+                            # take options (ex: boost) into consideration
+                            field_options = {key: value for key, value in _field.items() if key != 'name'}
                             field = "{}.{}".format(path, _field['name'])
+                            field_kwargs = {
+                                field: {
```
Comment (barseghyanartur) [inline src/django_elasticsearch_dsl_drf/filter_backends/search/query_backends/nested.py:82]: Look good, except that I don't want to drop Python 2.7 in a minor release just yet. Could you replase this with Python 2.7 compatible structure? Like, ``field_options.update({"query": search_term})`` and then ``field: field_options``.

Comment (hemache) [reply src/django_elasticsearch_dsl_drf/filter_backends/search/query_backends/nested.py]: @bashu done

**Comment by barseghyanartur** [comment]
@hemache:

Thanks!

Could you make another PR with updated tests/docs?

**Comment by hemache** [comment]
> Could you make another PR with updated tests/docs?

I'm not sure about the tests, couldn't find `NestedQueryBackend` test suite, can you please give a hint? 

it would be awesome if we can merge this PR first

**Pull request merged**